spec/factories/decks.rb

```diff
@@ -1,5 +1,5 @@
 FactoryGirl.define do
   factory :deck do
-    name 'Testes'
+    name 'Test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.